cilium/cmd/policy_wait.go

```diff
@@ -48,30 +48,44 @@ var policyWaitCmd = &cobra.Command{
 
 			needed := len(eps)
 			ready := 0
+			notReady := 0
 
 			for _, ep := range eps {
-				if ep.Policy != nil && ep.PolicyRevision >= reqRevision &&
-					ep.State == models.EndpointStateReady {
-					ready++
+				if ep.Policy != nil {
+					if ep.PolicyRevision >= reqRevision &&
+						ep.State == models.EndpointStateReady {
+						ready++
+					} else if ep.State == models.EndpointStateNotReady {
+						notReady++
+					}
 				}
 			}
 
 			if ready == needed {
-				if haveWaited {
+				if timeWaited > 0 {
 					fmt.Printf("\n")
 				}
 				return
+			} else if timeWaited > failWaitTime && notReady > 0 {
+				fmt.Printf("\n")
```

Comment:
Any particular reason why these newlines don't fold into the `Fatalf()` call?

Reply:
No. Had to check they go to the same stream and they do.

Reply:
I.e., folded, thanks!